docs/dynamic-plugins/installing-external-plugins.md

```diff
@@ -0,0 +1,181 @@
+# Installing External Plugins
+
+RHDH supports dynamic plugins, which are plugins not included in the core RHDH distribution. These plugins can be installed or uninstalled without rebuilding the RHDH application—only a restart is required to apply the changes.
+
+If your plugin not already packaged as a dynamic plugin, you must package it into one of the supported formats before installation.
+
+## Packaging a Backstage Plugin as a Dynamic Plugin
+
+To package a Backstage plugin as a dynamic plugin, you need access to its source code.
+
+First you need to create a derived package using the `@janus-idp/cli` and than package it into one of the supported formats.
+
+There are three possible packaging formats for dynamic plugins:
+
+- OCI image
+- `tgz` archive
+- JavaScript package
+
+**The OCI image is the recommended format.**
+
+The derived dynamic plugin JavaScript packages should **not** be pushed to the public npm registry. They should only be published to a private npm registry.
+
+### Step 1. Creating a Derived Dynamic Plugin Package
+
+In the root directory of the plugin source code, run the following command:
+
+```bash
+npx @janus-idp/cli@latest package export-dynamic-plugin
+```
+
+This will create a `dist-dynamic` directory, which contains the derived dynamic plugin package. This directory can then be packaged into one of the supported formats.
+
+### Step 2.a Packaging a Plugin into an OCI Image
+
+**Prerequisites:**
+
+- `podman` or `docker` installed on your system.
+- An exported derived dynamic plugin package (see: [Creating a Derived Dynamic Plugin Package](#step-1-creating-a-derived-dynamic-plugin-package).
+
+To package the plugin into an OCI image, use the `package package-dynamic-plugins` command from `@janus-idp/cli` in the plugin’s source code root directory (not in the `dist-dynamic` directory).
+
+The command will output the correct path definition for the plugin, which can be used in the `dynamic-plugin-config.yaml` file.
+
+```bash
+npx @janus-idp/cli@latest package package-dynamic-plugins --tag quay.io/example/image:v0.0.1
+```
+
+The `--tag` argument is required and specifies the image name and tag. The image won't be pushed to the registry automatically; use the `podman push` or `docker push` command to push the image to the registry.
```

Comment:
Maybe "The `--tag` argument is required when using this packaging method", only because I kinda might have  made it optional recently if you use `--export-to` for `package-dynamic-plugins`.